Cargo.toml

```diff
@@ -33,6 +33,7 @@ keywords = ["database"]
 default = ["linkage"]
 bundled = ["sqlite3-sys/bundled"]
 extension = []
+encryption = []
```

Comment:
Have you tried it? It should not work, as one has to enable `sqlite3-sys/encryption`.

Reply:
When I tried with with a test package, it does seem to react to it's absence. Admittedly, this is my first time seeing the Rust "-sys" pattern.

Reply:
OK, I will add it myself.